included warning box (#1474)
spryker/spryker-docs: docs/cloud/dev/spryker-cloud-commerce-os/preparation-for-going-live.md

  - /docs/en/preparation-for-going-live
---

> {% info_block warningBox "IMPORTANT: Do not risk your Go Live!" %}

Reviewer: ```suggestion
{% info_block warningBox "Do not risk your Go-Live!" %}
```

> The prepartion steps listed here are so critical to the success of your go live that we consider them mandatory. Please note that we expect every Spryker Customer and Partner to complete the list as described below. We will generally not be able to assist you with problems related to your go live that are caused by these steps not being completed in time. Please make sure that your project plan contains tasks related to the go live checklist and allows enough time for them to be completed. 

Reviewer: ```suggestion
The preparation steps listed here are mandatory because they are critical to the success of your go-live. We expect every Spryker customer and partner to complete the list as described below. We can't assist you with problems related to your go-live that are caused by these steps not being completed in time. Ensure that your project plan contains tasks related to the go-live checklist and allows enough time for them to be completed. 
```

  - /docs/en/preparation-for-going-live
---

{% info_block warningBox "Do not risk your Go-Live!" %}

> The preparation steps listed here are mandatory because they are critical to the success of your go-live. We expect every Spryker customer and partner to complete the list as described below. We can't assist you with problems related to your go-live that are caused by these steps not being completed in time. Ensure that your project plan contains tasks related to the go-live checklist and allows enough time for them to be completed. 

Reviewer 2: We cannot expect every customer to be on the latest versions of all modules, for example.

Other points in the document are also questionable, and I would suggest to split them into MUST and RECOMMENDED.

And this phrase `We can't assist you with problems related to your go-live` is a bit too hard.

Reviewer: @profuel, what point are you mentioning exactly?
@ThomasLehnerSpryker

Reviewer 2: I will talk to Thomas about this piece next year.

Reviewer: @profuel any updates on this PR?
